app/translations/en.json

```diff
@@ -205,7 +213,7 @@
     "title": "Receive",
     "title-syncing": "Receive - Syncing",
     "more": "more...",
-    "newu-option": "New Address",
+    "newu-option": "New Diversified Unified Address",
```

Comment:
We don't yet have the right underlying infrastructure to support diversification.  @AloeareV or @Oscar-Pepper needs to add support for Diversifier Introspection before we can expose new diversified addresses to a User.   If that support is not available in `zingolib`, then it will be impossible to know that we've recovered the full set of diversified addresses when Sprouting from a Seed.

Reply:
for now the only `new` UA is with: `tzo`, A full UA with the 3 receivers... I don't really know, but for me It's the same with one UA or several UA's...